doc/sphinx/source/howto/derive-for-training.rst

```diff
@@ -24,27 +25,64 @@ MNIST data.
   - :ref:`update`
 
 
+.. _understanding_ml_ecosystem:
+
+Understanding the ML ecosystem
+===============================
+
+In a :abbr:`Machine Learning (ML)` ecosystem, it makes sense to take advantage 
+of automatable tasks as much as possible. As such, nGraph was designed to use 
+all available "automatic" graph construction endpoints handed down to it from a 
```

Comment:
I would drop the qualifier "automatic" here, and maybe s/`handed down to it from a framework`/`provided by frameworks`/

Reply:
Done